src/chrome/content/rules/Orgmode.org.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Orgmode.org">
+	<target host="orgmode.org" />
+	<target host="beta.orgmode.org" />
+	<target host="code.orgmode.org" />
+	<target host="www.orgmode.org" />
```

Comment:
Can you move this target in second position please?

Reference: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering